python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
         pdf_arrow = df.toPandas()
+        # need to remove timezone for comparison
+        pdf_arrow["7_timestamp_t"] = \
+            pdf_arrow["7_timestamp_t"].apply(lambda ts: ts.tz_localize(None))
```

Comment:
Because of the way Spark handles timestamps, there needs to be a timezone in the Arrow data.  So when comparing `toPandas()` both ways, one needs to be changed.  I don't think it would be correct to add a timezone to the pandas.DataFrame created when `arrow.enable` is false.